### ISSUE TITLE
How to set k for top_k metric?

### ISSUE THREAD

**Issue by mrchypark** [rstudio/keras#190]
How to set k for top_k metric?

thank you in advance.

**Comment by jjallaire** [comment]
This is indeed not obvious! I just updated the docs with an example of how to do it:

https://keras.rstudio.com/reference/metric_binary_accuracy.html#examples

**Comment by mrchypark** [comment]
OMG. thank you!
